test/list.go

```diff
@@ -486,6 +489,130 @@ func DescribeListTestsFor(ctx *common.TestContext, t TestCase, responseMode Resp
 				})
 			}
 
+			if t.SupportsAsyncOperations {
+				Context("when attach_last_operations is truthy", func() {
+					var resourceWithOneOperation, resourceWithFewOperations common.Object
+					var lastOperationForResourceId string
+					var resourceWithOneOperationId, resourceWithFewOperationsId string
+
+					createTestOperation := func(resourceID string, opType types.OperationCategory) (types.Object, string) {
```

Comment:
why the complex return value? seems like only the operation ID is used

Reply:
yes just copied this from a different test, can be simplified